src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -168,6 +179,7 @@ export const generateContent: TranslationFn = content => {
     hasCertificateOfService,
     hasCertificateOfDeemedOrDispensedService,
     hasCertificateOfEntitlement,
+    isAwaitingAmendedApplication,
```

Comment:
might be more clear to call it isAwaitingAmendedApplicationState

Reply:
Hi @lara-mpt I updated the name to isAwaitingAmendedApplicationState.